.gitignore

```diff
@@ -16,3 +16,4 @@ html
 .rvmrc
 .ruby-version
 .ruby-gemset
+tags
```

Comment:
Hm, if you're generating ctags via an editor plugin, what do you think about putting this in a global gitignore in your home directory instead of per project?